config.rb

```diff
@@ -1,120 +1,25 @@
-# Ignore bower_components and assets directories when building
-ignore 'bower_components/*'
-ignore 'assets/*'
+ROOT_PATH = File.expand_path(File.dirname(__FILE__))
 
-# Add assets to sprockets import path
-ready do
-  root_path = File.expand_path(File.dirname(__FILE__))
-  sprockets.append_path "#{root_path}/source/assets/javascripts/"
-  sprockets.append_path "#{root_path}/source/assets/stylesheets/"
-end
-
-###
-# Compass
-###
+sprockets.append_path File.join root, 'bower_components'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.